src/mmda/predictors/heuristic_predictors/svm_word_predictor.py

```diff
@@ -0,0 +1,390 @@
+"""
+
+
+"""
```

Comment:
nit: suggest dropping the empty docstring or fleshing it out

Reply:
adding